scripts/upload-coverage.bat

```diff
@@ -0,0 +1,10 @@
+if not defined PYTEST_NO_COVERAGE (
+    echo Prepare to upload coverage information
+    C:\Python36\Scripts\pip install codecov
+    C:\Python36\Scripts\coverage combine
+    C:\Python36\Scripts\coverage xml
+    C:\Python36\Scripts\coverage report -m
+    C:\Python36\Scripts\codecov --required -X gcov pycov search -f coverage.xml --flags %TOXENV:-= %
```

Comment:
Also add a flag "win"/"windows" here always.

Reply:
Done (not pushed yet)